index.js

```diff
@@ -376,5 +378,27 @@ const testCommand = (arguments, receivedMessage) => {
   );
 };
 
+const verifyCommand = (arguments, receivedMessage) => {
+  if (arguments.length > 0){
+    receivedMessage.channel.send(
+      "Invalid arguments given."
+    );
+  }
+  else{
+    if (receivedMessage.member.roles.cache.some(r => r.name === "OIC" || r.name === "school-leads" || r.name === "temp_admin")){
```

Comment:
@Iscaraca would it be better if instead of using `r.name` we check for the roleID?